bin/run_tests.bash

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-docker-compose run --rm --entrypoint bash -c "RAILS_ENV='test' bundle exec rspec spec" app
+docker-compose run app bash -c "bundle exec rspec spec" --rm --entrypoint
```

Comment:
No, I think something changed in the CLI parser, because it used to work.

You changed my suggestion, and your new code works, but:

* The documentation says that `--entrypoint` requires an argument. In your command line, I think that `--entrypoint` is being ignored. I don't think it's necessary to use `--entrypoint`.
* The documentation says that options `--rm` and `--entrypoint` should be written after `run` but before `SERVICE`, which is probably why I had used the ordering that I did.

In case Docker fixes the parser, best to use the order that is specified in the documentation:

https://docs.docker.com/compose/reference/run/

```suggestion
docker-compose run --rm app bash -c "bundle exec rspec spec"
```

Reply:
Great thanks for checking for me :)